driver-core/src/test/java/com/datastax/driver/core/DataTypeIntegrationTest.java

```diff
@@ -61,6 +62,16 @@
         return statements;
     }
 
+    @AfterClass
+    @Override
+    public void afterClass() {
```

Comment:
Great idea.   I raised the timeout in clearKeyspace to 60 seconds, but I like the change better since it drops tables individually / probably cheaper.